pypeit/utils.py

```diff
@@ -55,6 +55,20 @@ def rebin(a, newshape):
     indices = coordinates.astype('i')  # choose the biggest smaller integer index
     return a[tuple(indices)]
 
+# JFH This function is only used by procimg.lacosmic. Can it be replaced by above?
```

Comment:
short doc string please

Reply:
I wrote the doc string for the function I created. This other rebin function was just renamed and had no doc string. My intention at present is not to add documentation everywhere, since we know that would be a Herculean task.